packages/mobile/src/tokens/saga.ts

```diff
@@ -46,7 +46,6 @@ export async function convertToContractDecimals(value: BigNumber, token: CURRENC
 
 export async function getTokenContract(token: CURRENCY_ENUM) {
   Logger.debug(TAG + '@getTokenContract', `Fetching contract for ${token}`)
-  await waitForWeb3Sync()
```

Comment:
@annakaz this was calling the waitForWeb3Sync saga from outside 'saga land'. It doesn't work and navigates the app to error after the sync timeout. Did we definitely need this here for something?

Reply:
Thanks for catching this. We have to ensure contractkit is connected to get the TokenContracts, but I just confirmed that we call `getConnectedAccount` every time before calling `getTokenContract`, so removing this is fine. Could be helpful to throw an error explaining this if it's called when the account isn't connected, but I can add this in another PR